build/pkgs/gap_packages/SPKG.rst

```diff
@@ -7,6 +7,22 @@ Description
 Several "official" and "undeposited" GAP packages available from
 https://www.gap-system.org/Packages/packages.html
 
+Installing this SPKG makes the corresponding GAP packages available to
+GAP within Sage, but unless Sage itself needs a given package, that
+package will not be loaded automatically. At the moment, only the
+PolyCyclic package meets that criterion. To load any other GAP
```

Comment:
I think "unless Sage itself needs a given package" needs a clarification. Aren't all the packages listed in the features used by "Sage itself"?

Reply:
The distinctionis between "used by" and "needed by." The packages in `all_features()` _can_ all be used by sage, but none are _required_ by sage. And of those "used by sage" packages, PolyCyclic is used especially heavily, which I thought was worthy of special treatment.

Reply:
I think that's not adequately expressed by the current wording

Reply:
Should be better now. I had forgotten (it was a whole day ago) that polycyclic is no longer part of gap_packages and is instead installed as part of the core gap SPKG. So now it's much easier to simply say that anything in gap_packages has to be loaded explicitly.

Reply:
Thanks. Much better.